dev/com.ibm.ws.security.openidconnect.clients.common/src/com/ibm/ws/security/openidconnect/backchannellogout/internal/JtiCacheKey.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.security.openidconnect.backchannellogout.internal;
+
+import java.util.Objects;
+
+import com.ibm.websphere.ras.annotation.Sensitive;
+
+class JtiCacheKey {
+
+    private final String jti;
+    private final String configId;
+
+    public JtiCacheKey(@Sensitive String jti, String configId) {
+        this.jti = jti;
+        this.configId = configId;
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(jti, configId);
+    }
```

Comment:
cache hashcode for better performance, since class is immutable

```suggestion
    private final String jti;
    private final String configId;
    
    private final int hashCode;

    public JtiCacheKey(@Sensitive String jti, String configId) {
        this.jti = jti;
        this.configId = configId;
    }

    @Override
    public int hashCode() {
        if (this.hashCode == 0) {
            this.hashCode = Objects.hash(jti, configId);
        }
        return this.hashCode;
    }
```